src/test_checker/mod.rs

```diff
@@ -70,6 +70,28 @@ impl TestChecker {
         Ok(path)
     }
 
+    fn check_expected_output_stream(&mut self, redirect: &Redirect, expected: Vec<u8>) -> R<()> {
+        match redirect.captured()? {
+            None => panic!(
+                "scriptkeeper bug: {} expected, but not captured",
+                redirect.stream_type
+            ),
+            Some(captured) => {
+                if captured != expected {
+                    self.register_error(format!(
+                        "  expected output to {}: {:?}\
```

Comment:
i realize this was moved from below, but the `trim_margin` function is nice if we can use it here, i think it makes things much more readable

Reply:
Sounds good. I moved the trim-margin dependency from the build dependencies list to general dependencies to make this work.